### DISCUSSION TITLE
First video upload on youtube results in success but youtube never processes the video.

### DISCUSSION THREAD

**Discussion by twilson90** [googleapis/google-api-nodejs-client#3611] [Q&A]
#### Environment details

  - OS: Windows 11
  - Node.js version: 22.9.0
  - npm version: 10.8.3
  - `googleapis` version: 140.0.1

Whenever I try to upload videos via the API, the first attempt always results in a video that never gets processed by YouTube.
The response from the API will indicate thyat the video has successfully uploaded, but in Youtube Studio I'll see:
![chrome_fk1xvFDjwe](https://github.com/user-attachments/assets/fc5304a3-8cd0-4b58-890d-64c63c87913a)

'Upload 0%' remains indefinitely (even though it's been completely uploaded) and it never gets processed.
I have to manually cancel it and run the script again, the second attempt will always succeed and the video will get processed.

Has anyone else experienced this behaviour? What might be the cause?

This is my script:
https://github.com/twilson90/concat-cam-videos/blob/main/index.js

**Comment by reutg** [comment]
Hi @twilson90, did you find the problem?

**Comment by twilson90** [comment]
Nope, still an issue.

**Comment by sofisl** [comment]
Since this is more specifically an API question, transferring this over to Discussions.
